documentation.html

```diff
@@ -493,12 +493,13 @@ <h3 class="title">Men Sweatshirt</h3>
         <!-- Responsive images  -->
         <section class="component-section" id="badges-section">
           <h4 class="component-heading">Images</h4>
-            <p class="component-subheading bold-text">
-              Example of responsive image </p>
-            <div class="component-example flex-row flex-row-center-vertical flex-gap-1 ">
-              <div style="width: 70%; height: 100% ; margin: 2rem 2rem 2rem 2rem;">
-                <img class=" img-responsive " src="./Assets/chatbot1.jpg" alt="bot">
-          </div></div>
+          <p class="component-subheading bold-text">
+            Example of responsive image </p>
+          <div class="component-example flex-row flex-row-center-vertical flex-gap-1 ">
+            <div style="width: 70%; height: 100% ; margin: 2rem 2rem 2rem 2rem;">
```

Comment:
you can remove this inline css, as it can as it isn;t a very good practice and even here you can use margin shorthand which is just margin: 2rem,